### PR TITLE
[Ewerton] - sage- Escrow service on Celo that holds funds until certain conditions are met

### PR DESCRIPTION
**Pull request by guizostudios**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [X] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [X] The title of the Pull Request matches the title of my tutorial.
- [X] I've followed the tutorial template structure.
- [X] I've proofread the tutorial for spelling and grammar mistakes.
- [X] I've thoroughly explained the code, with inline code comments and text explanations.
- [X] I've run a linter & prettifier to format my code blocks.
- [X] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+
+    ```solidity
+    function returnDaysForWithdrawal() public view returns(uint){
+        return daysForWithdrawal;
+    }
+    ```
+
+## Full Code
+
+    ```solidity
```
Comment (Julius170) [inline blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md:283]: I'm guessing the tab here was an accident 
remove the tab before the "```solidity", it's altering your code snippet from rendering properly

Comment (Julius170) [inline blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md:283]: I'm guessing the tab here was an accident 
remove the tab before the "```solidity", it's altering your code snippet from rendering properly

**Review by Julius170 — COMMENTED** [review]
hi @gconnect You have quite a cool piece here
just make the few changes and you'll be ready for merging
Although It's not being deployed, I've reviewed the reason and it's not from your end
Perhaps @viral-sangani can look into it for us

File: blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+---
+
+
+![header](../../src/data-tutorials/showcase/intermediate/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.png)
+
+
+## Prerequisite
+
+The [Remix IDE](https://remix-project.org/) is an open-source web and desktop application for creating and deploying Smart Contracts. Originally created for Ethereum, it fosters a fast development cycle and has a rich set of plugins with intuitive GUIs. Remix is used for the entire journey of contract development and is a playground for learning and teaching EVM-compatible blockchains like Celo.
+- [Build Smart Contract on Celo with Remix](https://docs.celo.org/blog/tutorials/6-steps-to-quickly-build-smart-contracts-on-celo-with-remix)
```
[the remark this answers is not in the document]
Comment (guizostudios) [reply blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md]: done

File: blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md (hunk 1 above)
Comment (guizostudios) [reply blog/03-07-2023-escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met/escrow-service-on-celo-that-holds-funds-until-certain-conditions-are-met.md]: done

**Review by guizostudios — COMMENTED** [review]
Done

**Pull request merged**